src/components/pages/ProfilePage.vue

```diff
@@ -383,8 +444,31 @@ export default {
         this.website = res.data.website;
         this.newsletter = res.data.newsletter;
         let countryFound = allCountries.find(country => country.name === this.country)
+
         if(countryFound)
           this.countryCode = countryFound['iso2'];
+
+        let profile_picture_url_from_server = res.data.profile_picture;
+
+        // If profile picture image is null or undefined, show default image.
+        if (profile_picture_url_from_server === undefined || profile_picture_url_from_server === null) {
+          // If current user, update profile picture state. If not, not update (so only picture in profile, and not in
+          // dropdown, is changed. In this case, we assign and show default image.
+          if (this.username_param === this.username) {
+            this.set_profile_picture_url({profile_picture_url: "https://upload.wikimedia.org/wikipedia/commons/2/2c/Default_pfp.svg"})
```

Comment:
I think this is ok for now, but in subsequent versions this should change.

The default profile picture for users that didn't upload a profile picture is currently hardcoded here, and is not in our server.

We should store it in our server to avoid problems (what if the original url of wikimedia is removed?), and assign the link in a common place for easier management.

Reply:
@AlbertoCasasOrtiz I suggest we do it now, otherwise we will forget. Can we add the image here instead? https://github.com/stanfordnmbl/opencap-viewer/tree/main/public/images

Reply:
Done!